.prow/api.yaml

```diff
@@ -51,3 +51,62 @@ presubmits:
               cpu: 2
             limits:
               memory: 6Gi
+  - name: pre-dashboard-api-unit
+    run_if_changed: "^(cmd|hack|pkg)/"
+    decorate: true
+    clone_uri: "ssh://git@github.com/kubermatic/dashboard.git"
+    labels:
+      preset-goproxy: "true"
+    spec:
+      containers:
+        - image: quay.io/kubermatic/build:go-1.19-node-18-2
```

Comment:
We recently updated to Go 1.19.3 for the KKP API so you'll have to switch these CI jobs to `quay.io/kubermatic/build:go-1.19-node-18-4`